gensrc/thrift/FrontendService.thrift

```diff
@@ -654,6 +654,25 @@ struct TSnapshotLoaderReportRequest {
     5: optional i32 total_num
 }
 
+enum TFrontendPingFrontendStatusCode {
+   OK = 0,
+   FAILED = 1
+}
+
+struct TFrontendPingFrontendRequest {
+   1: required i32 clusterId
```

Comment:
`required` field should be used very carefully,  reference [here](https://diwakergupta.github.io/thrift-missing-guide/#_defining_structs)

Reply:
I think if the filed is really required and the TFrontendPingFrontendRequest is firstly constructed to use, the field used requried is ok